cmip6_downscaling/methods/bcsd.py

```diff
@@ -1,6 +1,3 @@
-import os
-
-os.environ["PREFECT__FLOWS__CHECKPOINTING"] = "true"
```

Comment:
I will set this on the Prefect agent so we will not need this in individual modules anymore.